examples/gno.land/r/demo/gnofundme/gnofundme_test.gno

```diff
@@ -0,0 +1,65 @@
+package gnofundme
+
+import (
+	"std"
+	"testing"
+	"time"
+
+	"gno.land/p/demo/testutils"
+)
+
+func TestWithdraw(t *testing.T) {
+	// Setup
+	banktestAddr := std.DerivePkgAddr("gno.land/r/demo/gnofundme")
+	mainaddr := std.DerivePkgAddr("main")
+	std.TestSetOrigCaller(mainaddr)
+	std.TestSetOrigPkgAddr(banktestAddr)
+
+	// get and print balance of mainaddr.
+	// with the SEND, + 200 gnot given by the TestContext, main should have 300gnot.
+	banker := std.GetBanker(std.BankerTypeRealmSend)
+	mainbal := banker.GetCoins(mainaddr)
+	ctype := 0
+	title := "Campaign for Withdrawal"
+	description := "Description for Withdrawal"
+	goal := int64(1000)
+	begin := time.Now()
+	deadline := begin.Add(time.Second * 1)
+	ownerAddress := testutils.TestAddress("owner")
+	beneficiaryAddress := testutils.TestAddress("beneficiary")
+	std.TestSetOrigCaller(beneficiaryAddress)
+	NewUser("Beneficiary", "")
+	std.TestSetOrigCaller(ownerAddress)
+	NewUser("Owner", "")
+	NewCampaign(ctype, title, description, goal, begin.Unix(), deadline.Unix(), beneficiaryAddress)
```

Comment:
You can put most of these inside the `NewCampaign` function. You need to declare them only if you need to use them again (more than once) later

Reply:
4b765d9f

Reply:
[this commit](https://github.com/gnolang/gno/pull/2085/commits/4b765d9f37a36c2553e281aef2e4d38aae299c32)